src/lib/components/Swap/Input.tsx

```diff
@@ -39,6 +47,14 @@ export default function Input({ disabled }: InputProps) {
   // extract eagerly in case of reversal
   usePrefetchCurrencyColor(swapInputCurrency)
 
+  const isTradeLoading = useMemo(
+    () => TradeState.LOADING === tradeState || TradeState.SYNCING === tradeState,
+    [tradeState]
+  )
+  const independentField = useAtomValue(independentFieldAtom)
+  const isDependentField = independentField !== Field.INPUT
```

Comment:
nit: You could inline this if you wanted: `const isDependentField = Field.INPUT !== useAtomValue(independentFieldAtom)`